loader/src/main/java/net/neoforged/fml/loading/moddiscovery/ExplodedDirectoryLocator.java

```diff
@@ -31,12 +33,17 @@ public record ExplodedMod(String modid, List<Path> paths) {}
 
     @Override
     public List<IModLocator.ModFileOrException> scanMods() {
-        explodedMods.forEach(explodedMod ->
-                ModJarMetadata.buildFile(this,
-                        jar->jar.moduleDataProvider().findFile("/META-INF/mods.toml").isPresent(),
-                        null,
-                        explodedMod.paths().toArray(Path[]::new))
-                .ifPresentOrElse(f->mods.put(explodedMod, f), () -> LOGGER.warn(LogMarkers.LOADING, "Failed to find exploded resource mods.toml in directory {}", explodedMod.paths().get(0).toString())));
+        explodedMods.forEach(explodedMod -> {
+            var jarContents = new JarContentsBuilder().paths(explodedMod.paths().toArray(Path[]::new)).build();
+            if (jarContents.findFile("/META-INF/mods.toml").isPresent()) {
```

Comment:
Any reason this path has a leading slash?

Reply:
That's just copy/pasted from the deprecated methods :P